src/app/core/theme.component.scss

```diff
@@ -0,0 +1,19 @@
+.rect-btns-container {
```

Comment:
nazwa mało mówi, czemu tak to skracamy?

Reply:
kwestia indywidualnej preferencji :) zgadzam się, że jednak może być mało czytelne, zmieniłam na `rectangular-buttons-container`